templates/shrine/data/featured.json

```diff
@@ -4,8 +4,10 @@
     "title": "Green comfort chair",
     "quote": "Leave the tunnel and the rain is fallin amazing things happen when you wait",
     "imageUrl": "https://www.gstatic.com/angular/material-adaptive/shrine/chair.png",
+    "imageAlt": "foobar",
```

Comment:
I know but I would like to show it with different values. Because if you don´t do that (my opinion), maybe people missunderstand what should be the correct value of an `alt` attribute ([the context is important](http://webaim.org/techniques/alttext/)) and they think that it allways is valid with a title value you write in the page.